poetry.lock

```diff
@@ -183,13 +183,13 @@ testing = ["covdefaults (>=2.3)", "coverage (>=7.3)", "diff-cover (>=7.7)", "pyt
 
 [[package]]
 name = "identify"
-version = "2.5.27"
+version = "2.5.28"
```

Comment:
Did you run `poetry update` locally? Please remove unrelated changes from this pull request.

Reply:
Just `poetry lock` and `poetry install`. I'll remove the change though. Thanks for catching this.

Reply:
`poetry lock` runs `poetry update`, which is why [our pre-commit hook instead runs `poetry lock --no-update`](https://github.com/ScribeMD/pre-commit-hooks#poetry-lock) to prevent it from constantly performing lock file maintenance.

Reply:
Still the same problem as before.

Reply:
Oh darn. Sorry, just seeing this.

Reply:
Same problem again.